### PR TITLE
Fix hover nav positioning

### LINKED ISSUE #1834: Orange Ubuntu bar dropdowns unreachable on some pages
Can't click on the orange Ubuntu bar dropdowns on the 'Download Ubuntu Desktop' webpage. Other things about that top bar seem different (see the GIF below), the very top bar is one less pixel tall, the orange Ubuntu bar is taller, the bar below that is smaller, and the orange bar links are underlined as you hover over them.

Also applies to [the 'Get OpenStack' webpage](https://www.ubuntu.com/download/cloud).

Firefox 53.0.2 on Ubuntu 17.04.

peek 2017-05-25 01-52

---
*Reported from: https://www.ubuntu.com/download/desktop*

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Fix hover nav positioning

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/about](http://0.0.0.0:8001/about))
- Make sure the hover menu is in the correct position 

Fixes #1834 

[Demo](http://www.ubuntu.com-nav-fix.demo.haus/about)

### CODE REVIEW

**Comment by yaili** [comment]
- [ ] The hover-menu class should have no left padding - compare dropdowns in ubuntu.com live homepage and this demo/new Vanilla pages - there's too much left padding

**Comment by pmahnke** [comment]
@grahambancroft @yaili I am pushing this live... it is critical, it should go live ASAP.

File: static/sass/_v1_pattern_navigation.scss [hunk 1]
```diff
@@ -258,7 +258,7 @@
     }
 
     li {
-      padding: $sp-x-small $sp-small 0;
+      padding: $sp-x-small $sp-small 0 0;
```
Comment (pmahnke) [inline static/sass/_v1_pattern_navigation.scss]: removed padding-left per @yaili

**Comment by anthonydillon** [comment]
This branch was merged with a -1 label.

**Pull request merged**